rules_csharp_gapic/csharp_gapic_repo.bzl

```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def _gapic_generator_src_impl(ctx):
+    ctx.download_and_extract(
+        url = "https://github.com/googleapis/gapic-generator-csharp/archive/v1.0.0-beta06.tar.gz",
```

Comment:
These arguments look like they should be on the sruface of the rule itself, not hardcoded in the implementation (otherwise we would need to update the implementation for each release of gapic-generator-csharp).